engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages.html

```diff
@@ -41,81 +42,45 @@ <h4 translate>Package Actions</h4>
   </section>
 </section>
 
-<section>
+<div data-extend-template="layouts/details-nutupane.html">
   <h4 translate>Installed Packages</h4>
 
-  <div class="row">
-    <form>
-      <input type="text"
-             class="form-control filter-input"
-             placeholder="{{ 'Filter' | translate }}"
-             ng-model="currentPackagesTable.filter"/>
-    </form>
-    <span class="nutupane-info" translate>Showing {{ currentPackagesTable.resource.total }} Packages</span>
-
-    <div class="fr" ng-hide="denied('edit_content_hosts', contentHost)" bst-feature-flag="remote_actions">
-       <button class="btn btn-primary"
-               translate
-               ng-click="updateAll()"
-               ng-disabled="working">
-         Update All
-       </button>
+  <span data-block="no-rows-message" translate>
+    You currently do not have any Packages installed. You can use Package Actions above this message to install new Packages.
```

Comment:
I might would say something more along the lines of 

"The host has not reported any installed packages, registering with subscription-manager should cause these to be reported."